### DISCUSSION TITLE
How to disable span, e.g. new_established_connection

### DISCUSSION THREAD

**Discussion by gftea** [libp2p/rust-libp2p#5443] [Q&A]
I tried out to disable it using RUST_LOG=[new_established_connection]=off,  but it does not work.
This trace comes from different libp2p modules, and the parent target is set to Span::none().

> [2024-06-04T14:20:13.146668Z+00:00] DEBUG new_established_connection{remote_addr=/ip4/127.0.0.1/tcp/25000/p2p/12D3KooWA4uuPr3e24mz82KLzAf2HW9pQAp4KuN9xpvub4sCaHFv id=3 peer=12D3KooWA4uuPr3e24mz82KLzAf2HW9pQAp4KuN9xpvub4sCaHFv}:Connection::poll: libp2p_swarm::connection: start + duration cannot be presented, halving duration start=Instant { tv_sec: 634346, tv_nsec: 280518583 } duration=18446744073709551615s
> [2024-06-04T14:20:13.146672Z+00:00] DEBUG new_established_connection{remote_addr=/ip4/127.0.0.1/tcp/25000/p2p/12D3KooWA4uuPr3e24mz82KLzAf2HW9pQAp4KuN9xpvub4sCaHFv id=3 peer=12D3KooWA4uuPr3e24mz82KLzAf2HW9pQAp4KuN9xpvub4sCaHFv}:Connection::poll: libp2p_swarm::connection: start + duration cannot be presented, halving duration start=Instant { tv_sec: 634346, tv_nsec: 280518583 } duration=9223372036854775807.5s
> [2024-06-04T14:20:13.146677Z+00:00] DEBUG new_established_connection{remote_addr=/ip4/127.0.0.1/tcp/25000/p2p/12D3KooWA4uuPr3e24mz82KLzAf2HW9pQAp4KuN9xpvub4sCaHFv id=3 peer=12D3KooWA4uuPr3e24mz82KLzAf2HW9pQAp4KuN9xpvub4sCaHFv}:Connection::poll: yamux::connection::rtt: sending ping 1095198323    
> [2024-06-04T14:20:13.146771Z+00:00] DEBUG new_established_connection{remote_addr=/ip4/127.0.0.1/tcp/25000/p2p/12D3KooWA4uuPr3e24mz82KLzAf2HW9pQAp4KuN9xpvub4sCaHFv id=3 peer=12D3KooWA4uuPr3e24mz82KLzAf2HW9pQAp4KuN9xpvub4sCaHFv}:Connection::poll: multistream_select::negotiated: Negotiated: Received confirmation for protocol: /yamux/1.0.0    
> [2024-06-04T14:20:13.146858Z+00:00] DEBUG new_established_connection{remote_addr=/ip4/127.0.0.1/tcp/25000/p2p/12D3KooWA4uuPr3e24mz82KLzAf2HW9pQAp4KuN9xpvub4sCaHFv id=3 peer=12D3KooWA4uuPr3e24mz82KLzAf2HW9pQAp4KuN9xpvub4sCaHFv}:Connection::poll: yamux::connection::rtt: received pong 1095198323, estimated round-trip-time 180.875µs    
>

**Comment by jxs** [comment]
are you sure that syntax using square brackets is valid?  Tracing follows [`env_logger`](https://docs.rs/env_logger/latest/env_logger/#enabling-logging) syntax.
Either that or you are not naming correctly the module you wanna silence.

**Comment by gftea** [comment]
in fact, the target is `yamux::connection::rtt` for tracing instrument, it is not a normal env trace format
